src/VsixV3/ProjectSystemPackage/Microsoft.VisualStudio.ProjectSystem.Managed.CommonFiles.swixproj

```diff
@@ -14,7 +14,7 @@
   <Import Project="$(NuGetPackageRoot)\MicroBuild.Core\$(MicroBuildVersion)\build\MicroBuild.Core.props" />
 
   <PropertyGroup>
-    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);Version=$(AssemblyVersion);OutputPath=$(OutDir);SourcePath=$(OutDir)\..\..\src;LicenseUri=http://www.microsoft.com</PackagePreprocessorDefinitions>
+    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);Version=$(BuildVersion);OutputPath=$(OutDir);SourcePath=$(OutDir)\..\..\src;LicenseUri=http://www.microsoft.com</PackagePreprocessorDefinitions>
```

Comment:
Is BuildVersion greater than what is currently shipping? Does it pick the right scheme (2.0.0 vs 15.0.0) for CommonFiles vs Editors?